Links added for Open Source simulators (#2794)
cocotb/cocotb: documentation/source/simulator_support.rst

    make SIM=icarus

.. note::

>     This section requires `installation <https://iverilog.fandom.com/wiki/Installation_Guide>`_ of `Icarus Verilog <http://iverilog.icarus.com/>`_.

Reviewer: "This section" sounds a bit odd to me: it's not this section of text that needs Icarus Verilog, but running a design with it. How about (something like) "A working installation of Icarus Verilog is required. Refer to their documentation for installation instructions."

>     A working installation of `Icarus Verilog <http://iverilog.icarus.com/>`_ is required. You can find installation instructions `here <https://iverilog.fandom.com/wiki/Installation_Guide>`_.

Reviewer 2: ```suggestion
    A working installation of `Icarus Verilog <http://iverilog.icarus.com/>`_ is required.
    You can find installation instructions `here <https://iverilog.fandom.com/wiki/Installation_Guide>`_.
```

    make SIM=verilator

.. note::

>     A working installation of `Verilator <https://www.veripool.org/verilator/>`_ is required. You can find installation instructions can  `here <https://verilator.org/guide/latest/install.html>`_.

Reviewer 2: ```suggestion
    A working installation of `Verilator <https://www.veripool.org/verilator/>`_ is required.
    You can find installation instructions `here <https://verilator.org/guide/latest/install.html>`_.
```

> 

Reviewer 2: ```suggestion
```

    make SIM=ghdl

.. note::

>     A working installation of `GHDL <http://ghdl.free.fr/>`_ is required. You can find installation intructions `here <http://ghdl.free.fr/site/pmwiki.php?n=Main.Installation>`_.

Reviewer 2: ```suggestion
    A working installation of `GHDL <https://ghdl.github.io/>`_ is required. You can find installation instructions `here <https://ghdl.github.io/ghdl/getting.html>`_.
```

> 

Reviewer 2: ```suggestion
```
